pkg/config/expand.go

```diff
@@ -593,7 +593,19 @@ func expandSimpleVariable(
 func expandVariable(
 	context varContext,
 	modToGrp map[string]int) (string, error) {
-	return "", fmt.Errorf("%s: expandVariable", errorMessages["notImplemented"])
+	re := regexp.MustCompile(anyVariableExp)
+	matchall := re.FindAllString(context.varString, -1)
+	errHint := ""
+	for _, element := range matchall {
+		// get first 2 chars
```

Comment:
Summary of offline discussion... functionality is right. Let's improve commenting so future archaeologists can understand the regular expression and its behavior. Change:

```
// get first 2 chars
```

to something like

```
the regex match will include the first matching character
this might be (1) "^" or (2) any character EXCEPT "\"
if (2), we have to remove the first character from the match
```